packages-exp/auth-exp/src/core/persistence/in_memory.test.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { inMemoryPersistence as persistence } from './in_memory';
+import { PersistenceType } from '.';
+import { expect } from 'chai';
+import { User } from '../../model/user';
+import { testUser } from '../../../test/mock_auth';
+
+describe('core/persistence/in_memory', () => {
```

Comment:
should we standardize on how we name these?

Reply:
Sure, how would you like to standardize them? I don't really care which way we end up choosing

Reply:
me neither, is there a pattern that the other packages follow? @Feiyang1